src/scss/aruba/_aruba.defaults.scss

```diff
@@ -19,7 +19,7 @@ $brand-status-colors: (
   disabled: #999999
 );
 $brand-link-color: $brand-color;
-$brand-grey-colors: (#2d3034, #52555a, #999999);
+$brand-grey-colors: (#2d3034, #52555a, #999999, #666666);
```

Comment:
@ericsoderberghp I removed the added grey color from aruba and resolved the build step error from gulp dist